File: 250319/20대의 평균 나이/average-age-of-20.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int sum = 0;
        int count = 0;
        double avg = 0;
        while(true) {
            int n = sc.nextInt();
            if(n >= 30 && n < 20) {
                break;
            }
            sum += n;
            count++;
        }
        avg = (double)sum / count;
        System.out.printf("%.2f", avg);
        sc.close();
    }
}

File: 250319/20대의 평균 나이/average-age-of-20.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int sum = 0, count = 0;
    double avg = 0;
    while (true)
    {
        int n = 0;
        cin >> n;
        if(n >= 30 || n < 20) {
            break;
        }
        else {
            sum += n;
            count++;
        }
    }
    cout << fixed;
    cout.precision(2);
    avg = (double)sum / count;
    cout << avg;
    return 0;
}